### DISCUSSION TITLE
Opensprinkler

### DISCUSSION THREAD

**Discussion by etamtlosz** [tteck/Proxmox#1060] [Ideas]
I'd like to recommend opensprinkler. It would be nice to see it between the other helper scripts.

**Comment by nagyrobi** [comment]
How exactly do you think opensprinkler would be useful as a script?

**Reply by etamtlosz** [reply]
Robi,
as I understand all the scripts provided by tteck are called "Proxmox Helper Scripts", that's why I called it as such.
TBH I tried installing opnsprinkler as a container in proxmox, a VM, and I was struggling to make it work. Right now I have it in a docker container.

**Reply by nagyrobi** [reply]
> I'd like to recommend opensprinkler. It would be nice to see it between the other helper scripts.

Are we talking about the same? [opensprinkler](https://opensprinkler.com/) is an ESP-based irrigation controller hardware, with open sourced firmware (binaries also [available](http://raysfiles.com/os_compiled_firmware/)), which can rely on cloud based weather data to control watering levels. Which part do you want to have running in an LXC container?

**Reply by etamtlosz** [reply]
That's the opensprinkler I'm talking about. Right now I am using [this](https://hub.docker.com/r/mciupak/opensprinkler) docker image on my qnap.
I have a SONOFF 4CH Pro R3 flashed with esphome, which is scheduled/controlled via the opensprinkler running in docker.
I know I could automate it via HA, but doing it in the opensprinkler UI it is easier and more faster, since it has wx forecast and other stuff integrated.

**Reply by nagyrobi** [reply]
Oh... The unofficial port...

**Reply by etamtlosz** [reply]
I didn't find any other way to get it working. Since it is an OS, I thought it could be ported (somehow?!) to proxmox as a CT.

**Reply by tteck** [reply]
I _think_ it can
https://github.com/rszimm/sprinklers_pi/wiki
